bin/test_travis.sh

```diff
@@ -119,9 +119,15 @@ fi
 if [[ "${TEST_SYMPY}" == "true" ]]; then
     cat << EOF | python
 print('Testing SYMPY, split ${SPLIT}')
-import sympy
-if not sympy.test(split='${SPLIT}'):
+import sympy, coverage
+cov = coverage.Coverage()
+cov.start()
+if not sympy.test(split='${SPLIT}', subprocess=False):
```

Comment:
Can someone comment on this subprocess argument? As far as I can tell coverage.py needs the code to be run in the main process and thus this flag is required. I don't understand what the hash randomization is and how the subprocess flag affects it.